code/app/src/androidTest/java/com/example/goldencarrot/FacilityUserImplTest.java

```diff
@@ -0,0 +1,57 @@
+package com.example.goldencarrot;
+
+import com.example.goldencarrot.data.model.user.FacilityUserImpl;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.Optional;
```

Comment:
This test is in the wrong path.

Please move this Unit test to code/app/src/test/java/com/example/goldencarrot

Reply:
We don't really need this test file anymore so I will just get rid of it for now.